src/main/java/slimeknights/tconstruct/smeltery/multiblock/MultiblockCuboid.java

```diff
@@ -71,6 +71,11 @@ public MultiblockStructure detectMultiblock(World world, BlockPos center, int li
       }
     }
 
+    // below lowest internal position
```

Comment:
Good analysis and correct fix. However you should move the check up directly after the center is calculated (Line 40).

Reply:
Good point, Done.